### PR TITLE
feat: button group basic

### LINKED ISSUE #201: Add `Button Groups` examples
### Subject

`Button Groups`

### Description

We need to make `Button Groups` examples. You may also refer to https://tailwindui.com/components/application-ui/elements/button-groups when making these examples.

### Are you willing to participate in the development of this feature and create a pull request?

None

### PR DESCRIPTION
**Pull request by nanikasi**
Closes #201 

## Description
Add ButtonGroup sample

### CODE REVIEW

**Review by Hayato1031 — DISMISSED** [review]

File: contents/forms/button-group/basic/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "en": {
+    "title": "Basic",
+    "description": ""
```
Comment (cidkumagai) [inline contents/forms/button-group/basic/metadata.json:4]: Please write a description in English.

File: contents/forms/button-group/basic/metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "ja": {
+    "title": "ベーシック",
+    "description": ""
```
Comment (cidkumagai) [inline contents/forms/button-group/basic/metadata.json:8]: Please write a description in Japanese.

File: contents/forms/button-group/metadata.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "en": {
+    "title": "ButtonGroup",
+    "description": ""
```
Comment (cidkumagai) [inline contents/forms/button-group/metadata.json:4]: Please write a description in English.

File: contents/forms/button-group/metadata.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+  },
+  "ja": {
+    "title": "ボタングループ",
+    "description": ""
```
Comment (cidkumagai) [inline contents/forms/button-group/metadata.json:8]: Please write a description in Japanese.

**Review by cidkumagai — CHANGES_REQUESTED** [review]

**Review by zonoryo17 — CHANGES_REQUESTED** [review]
Please correct the error in CI.

File: contents/forms/button-group/basic/metadata.json (hunk 1 above)
Comment (nanikasi) [reply contents/forms/button-group/basic/metadata.json]: Thank you for your review!

**Review by cidkumagai — APPROVED** [review]
LGTM!

File: contents/forms/button-group/basic/index.tsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import { Button, ButtonGroup } from "@yamada-ui/react"
+import type { FC } from "react"
+
+const Basic: FC = () => {
```
Comment (zonoryo17) [inline contents/forms/button-group/basic/index.tsx:4]: @nanikasi 
badge
badge

I think `"BasicButtonGroup"` is clearer as a naming convention, what do you think?

Comment (nanikasi) [reply contents/forms/button-group/basic/index.tsx:4]: @zonoryo17 
Thank you for your review. The reason I used "basic" as the name is that I followed TailwindCSS's example from this link: https://tailwindui.com/components/application-ui/elements/button-groups. However, "BasicButtonGroup" is clearer compared to other components. I will change "Basic" to "BasicButtonGroup" and also update the directory name accordingly. Thank you.

**Review by zonoryo17 — APPROVED** [review]
It looks good to me.

**Review by hirotomoyamada — APPROVED** [review]

**Comment by hirotomoyamada** [comment]
Thanks!

**Pull request merged**